### PR TITLE
test: validate manual-approval-gate version

### PR DESCRIPTION
**Pull request by srivickynesh**

### CODE REVIEW

File: pkg/tkn/tkn.go [hunk 1]
```diff
@@ -41,6 +41,8 @@ func AssertComponentVersion(version string, component string) {
 		actualVersion = cmd.MustSucceed("oc", "get", "tektonhub", "hub", "-o", "jsonpath={.status.version}").Stdout()
 	case "results":
 		actualVersion = cmd.MustSucceed("oc", "get", "tektonresult", "result", "-o", "jsonpath={.status.version}").Stdout()
+	case "manual-approval-gate":
+		actualVersion = cmd.MustSucceed("oc", "get", "manualapprovalgate", "manualapprovalgate", "-o", "jsonpath={.status.version}").Stdout()
```
Comment (ppitonak) [inline pkg/tkn/tkn.go:45]: BTW, the name of resource must be `manual-approval-gate`, when you create a CR with different name you get a message like this
```
Ready: Resource ignored, Expected Name: manual-approval-gate, Got Name: manual-approval-gate-foo
```

Comment (srivickynesh) [reply pkg/tkn/tkn.go]: where should I place this CRD?

File: specs/olm.spec [hunk 2]
```diff
@@ -19,6 +19,11 @@ Steps:
     | S.NO | resource_dir                   |
     |------|--------------------------------|
     | 1    | testdata/hub/tektonhub.yaml    |
+  * Verify namespace "openshift-pipelines" exists
```
Comment (ppitonak) [inline specs/olm.spec:22]: this is not necessary, it's already present  above

**Review by ppitonak — APPROVED** [review]

**Review by ppitonak — APPROVED** [review]

**Pull request merged**